README.md

```diff
@@ -182,7 +182,7 @@ __Recommendation:__ You typically won't want to change this.
 __Optional__
 
 If you are running multiple server processes then you should make sure every one has a unique `worker` id. Should be an integer between 0 and 16. 
-If you do not do this there is very little chance of two servers generating the same id, but it is theatrically possible 
+If you do not do this there is very little chance of two servers generating the same id, but it is theoretically possible 
```

Comment:
Since the readme is generated, you'll want to update `templates/readme/api.md` as well.  Don't worry about re-running the generator, just keep it in sync or kill the `templates` directory and grunt task and not worry about it anymore.